This does not build with ImpactX:
```
/home/axel/src/amrex/Src/Base/AMReX_Math.H:162:26: error: template-id 'sincos<>' for 'std::pair<double, double> amrex::Math::sincos(double)' does not match any template declaration
  162 | std::pair<double,double> sincos (double x)
      |                          ^~~~~~
/home/axel/src/amrex/Src/Base/AMReX_Math.H:145:26: note: candidate is: 'template<class T_Real, typename std::enable_if<is_simd_v<T_Real>, int>::type <anonymous> > std::pair<_FIter, _FIter> amrex::Math::sincos(T_Real)'
  145 | std::pair<T_Real,T_Real> sincos (T_Real x)
      |                          ^~~~~~
/home/axel/src/amrex/Src/Base/AMReX_Math.H:177:24: error: template-id 'sincos<>' for 'std::pair<float, float> amrex::Math::sincos(float)' does not match any template declaration
  177 | std::pair<float,float> sincos (float x)
      |                        ^~~~~~
/home/axel/src/amrex/Src/Base/AMReX_Math.H:145:26: note: candidate is: 'template<class T_Real, typename std::enable_if<is_simd_v<T_Real>, int>::type <anonymous> > std::pair<_FIter, _FIter> amrex::Math::sincos(T_Real)'
  145 | std::pair<T_Real,T_Real> sincos (T_Real x)
      |                          ^~~~~~
/home/axel/src/amrex/Src/Base/AMReX_Math.H:216:26: error: template-id 'sincospi<>' for 'std::pair<double, double> amrex::Math::sincospi(double)' does not match any template declaration
  216 | std::pair<double,double> sincospi (double x)
      |                          ^~~~~~~~
/home/axel/src/amrex/Src/Base/AMReX_Math.H:198:26: note: candidate is: 'template<class T_Real, typename std::enable_if<is_simd_v<T_Real>, int>::type <anonymous> > std::pair<_FIter, _FIter> amrex::Math::sincospi(T_Real)'
  198 | std::pair<T_Real,T_Real> sincospi (T_Real x)
      |                          ^~~~~~~~
/home/axel/src/amrex/Src/Base/AMReX_Math.H:231:24: error: template-id 'sincospi<>' for 'std::pair<float, float> amrex::Math::sincospi(float)' does not match any template declaration
  231 | std::pair<float,float> sincospi (float x)
      |                        ^~~~~~~~
/home/axel/src/amrex/Src/Base/AMReX_Math.H:198:26: note: candidate is: 'template<class T_Real, typename std::enable_if<is_simd_v<T_Real>, int>::type <anonymous> > std::pair<_FIter, _FIter> amrex::Math::sincospi(T_Real)'
  198 | std::pair<T_Real,T_Real> sincospi (T_Real x)
      |                          ^~~~~~~~
gmake[2]: *** [_deps/localablastr-build/_deps/localamrex-build/Src/CMakeFiles/amrex_3d.dir/build.make:79: _deps/localablastr-build/_deps/localamrex-build/Src/CMakeFiles/amrex_3d.dir/Base/AMReX_BlockMutex.cpp.o] Error 1
gmake[1]: *** [CMakeFiles/Makefile2:2427: _deps/localablastr-build/_deps/localamrex-build/Src/CMakeFiles/amrex_3d.dir/all] Error 2
gmake: *** [Makefile:146: all] Error 2
```

--- a/Src/Base/AMReX_Math.H
+++ b/Src/Base/AMReX_Math.H
@@ -134,11 +134,9 @@ namespace detail {
 }
 
 //! Return sine and cosine of given number
-template<typename T_Real,
-#ifdef AMREX_USE_SIMD
-         std::enable_if_t<std::experimental::is_simd_v<T_Real>,int> = 0
-#else
-         std::enable_if_t<std::is_floating_point_v<T_Real>,int> = 0
+template<typename T_Real
+#ifndef AMREX_USE_SIMD
+         , std::enable_if_t<std::is_floating_point_v<T_Real>,int> = 0
 #endif
          >
 AMREX_GPU_HOST_DEVICE AMREX_FORCE_INLINE
@@ -187,11 +185,9 @@ std::pair<float,float> sincos (float x)
 }
 
 //! Return sin(pi*x) and cos(pi*x) given x
-template<typename T_Real,
-#ifdef AMREX_USE_SIMD
-         std::enable_if_t<std::experimental::is_simd_v<T_Real>,int> = 0
-#else
-         std::enable_if_t<std::is_floating_point_v<T_Real>,int> = 0
+template<typename T_Real
+#ifndef AMREX_USE_SIMD
+         , std::enable_if_t<std::is_floating_point_v<T_Real>,int> = 0
 #endif
          >
 AMREX_GPU_HOST_DEVICE AMREX_FORCE_INLINE

